tasks/00_ingest/post.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+from common import app_config, sql
+import sys
+import pymysql
+
+
+if __name__ == '__main__':
+    """Confirm database meets expectations."""
+
+    error = False
+    count = 0
+
+    with sql.db_cnx() as cnx:
+        with cnx.cursor() as c:
+            query = f"""
+                    SELECT count(*)
+                    FROM {app_config.schema}.{app_config.cases_raw};
+                    """
+
+            try:
+                c.execute(query)
+                count = c.fetchone()[0]
+                if count == 0:
+                    error = True
+                    print(f'Expected {app_config.cases_raw}',
+                          'table to be populated,',
+                          'found 0 records')
+            except pymysql.err.ProgrammingError as e:
```

Comment:
it could be useful to catch multiple exception types here, something like:
```
except (pymysql.err.ProgrammingError, pymysql.err.OperationalError) as e:
    ...etc...
```

i think that would probably cover the [error types](https://github.com/PyMySQL/PyMySQL/blob/main/pymysql/err.py) we're most likely to encounter here (and could be incorporated into other db-connection heavy scripts) -- what do you think?

Reply:
(in general, we should probably rethink exception handling in the scripts that do a lot of DB operations, but it might make sense to do that in the set of changes needed to switch to postgres, since i think the psycopg2 error classes are slightly different.)